setup.py

```diff
@@ -192,8 +192,8 @@ def read(*names, **kwargs):
         'dparse >= 0.4.1',
 
         # used to fix mojibake in Windows PE
-        'ftfy <  5.0.0; python_version == "2.7"',
-        'ftfy>=  5.0.0; python_version > "3"',
```

Comment:
But my github pypi contains this package.

Reply:
That's OK, we can have newer and older versions alright there.